maintainers/maintainer-list.nix

```diff
@@ -1483,6 +1483,12 @@
     githubId = 33503784;
     name = "Yucheng Zhang";
   };
+  cheriimoya = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add cheriimoya``.

Reply:
missed that one